scripts/test-fallback-scsv.py

```diff
@@ -36,24 +38,28 @@ def help_msg():
     print(" -e probe-name  exclude the probe from the list of the ones run")
     print("                may be specified multiple times")
     print(" --tls-1.3      server does support TLS 1.3")
+    print(" -d             negotiate (EC)DHE instead of RSA key exchange")
     print(" --help         this message")
 
 
 def main():
     host = "localhost"
     port = 4433
     run_exclude = set()
+    dhe = False
     tls13 = False
 
     argv = sys.argv[1:]
-    opts, args = getopt.getopt(argv, "h:p:e:", ["help", "tls-1.3"])
+    opts, args = getopt.getopt(argv, "h:p:e:d", ["help", "tls-1.3"])
     for opt, arg in opts:
         if opt == '-h':
             host = arg
         elif opt == '-p':
             port = int(arg)
         elif opt == '-e':
             run_exclude.add(arg)
+        elif opt == '-d':
```

Comment:
Why test it with a flag and not with a separate test? Also seeing below ECDHE and DHE cannot be tested separately, ECDHE will only be tested by a server that supports both.